src/app/views/apps/modals/app-settings-modal.html

```diff
@@ -26,13 +26,23 @@
           </div>
         </div>
       </div>
-
       <!-- Add on settings -->
       <div ng-show="helper.isAddOnSettingShown(app)">
         <hr>
         <button ng-click="helper.addOnSettingLauch(app)" class="btn btn-warning">{{ 'mno_enterprise.templates.impac.dock.settings.access_add-on_settings' | translate }}</button>
       </div>
-
+      <!-- Billing Details -->
+      <div ng-show="helper.canDisplayBilling()">
+        <hr>
+        <div ng-show="!!app.freeTrialEndAt">
+            Free Trial: {{ app.freeTrialEndAt }}
+        </div>
+        <div ng-show="!app.freeTrialEndAt && app.perUserLicence">
```

Comment:
Where are `app.freeTrialEndAt` and `app.perUserLicence` defined?
Are they supposed to come from the API? (In this case the variables would be snake cased)